### DISCUSSION TITLE
How to use @unocss/reset with SvelteKit?

### DISCUSSION THREAD

**Discussion by dhalucario** [unocss/unocss#2042] [Q&A] [answered]
Hi, I was wondering where I'd put my CSS resets in SvelteKit when wanting to use `@unocss/reset`.

**Comment by kamontat** [comment]
You can see how to configure sveltekit here: https://github.com/unocss/unocss/tree/main/packages/vite#sveltekit

**Reply by dhalucario** [reply]
I see how to configure SvelteKit. This doesn't fix my issue though. normalize.css still doesn't load after settings this up since this isnt a ruleset.

**Comment by leddy231** [comment, marked as answer]
I put mine in my root +layout.svelte

```
<script lang="ts">
    import 'uno.css';
    import '@unocss/reset/tailwind.css';
    ...
</script>
```
